### ISSUE TITLE
Size of the library

### ISSUE THREAD

**Issue by ronyrun** [angular-ui/angular-google-maps#1764]
Hello,

I'm using a cordova application with this lib and the command : bower install angular-google-maps

Install a package of 3,01 MB.
Is there a way to reduce the size (with maybe getting rid of the map files in the bower lib)?

This is impacting strongly the cordova mobile app size

Regards, 
Ron

**Comment by nmccready** [comment]
The library will need to be broken apart to make the components optional. Specifically some of the google maps extensions.

Examples:
- custering
- infoBox
- richmarker
- markerwlabel
- OverlappingMarkerSpiderfier

Along with that the extensions / overrides for those would need to be optional.

Other things:
- remove uuid
- dead code
- and I am sure there is more

**Comment by nmccready** [comment]
Also, in all honesty I do not recommend angularjs for mobile anyways. But in the end the above will help.

**Comment by ronyrun** [comment]
I'm using Ionic which is build on AngularJS?
It works :)

**Comment by nmccready** [comment]
Yeah it works; not saying it won't. Anyway I am giving advice on how to split this module. I don't really have time to do this myself right now. So yes, I am looking for help here.

**Comment by shyamal890** [comment]
+1
